DEVELOPMENT.md

```diff
@@ -0,0 +1,101 @@
+# Run services with `go run`
+
+This section explains how to run some of the services using `go run` instead of the Docker setup to facilitate development.
+
+## Running `sda-download` with `go run`
+
+- Bring up all SDA services with the S3 backend and populate them with test data by running the following command in the root folder of the repository:
+
+```sh
+make integrationtest-sda-s3-run 
```

Comment:
This command will build everything and run the tests and in the end will remove all containers.
So, there won't be anything running to test with.

Reply:
The command `make integration-sda-s3` will remove all containers when it is finished, but not this one 😄 
see https://github.com/neicnordic/sensitive-data-archive/blob/main/Makefile#L118-L119

Reply:
Ooh, right! Should have read more carefully :blush: :